src/Microsoft.AspNetCore.SignalR.Common/IHubProxy.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Threading.Tasks.Channels;
+
+namespace Microsoft.AspNetCore.SignalR
+{
+    public interface IHubProxy
+    {
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="methodName"></param>
+        /// <param name="args"></param>
+        Task Invoke(string methodName, params object[] args);
```

Comment:
Not sure I like this. It throws me off completely, as seen by my earlier comments about the naming. The name is just totally wrong. I'm not yet convinced that changing the name of this would be any worse than the other breaking changes we're making (heck, it's probably easier to resolve than any of those, just find/replace) and would mean we could make it something clearer like `SendAsync`, which I would expect to resolve when a message is **sent** vs. `InvokeAsync` which to me implies that it resolves when the underlying method finishes.